app/graphql/mutations/send_question.rb

```
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Mutations
+  class SendQuestion < Mutations::BaseMutation
+    argument :question_id, ID, required: true
+
+    field :question, Types::QuestionType, null: false
+
+    def resolve(question_id:)
+      unless Question.exists?(id: question_id)
+        return GraphQL::ExecutionError.new("Question with id #{question_id} does not exist.")
+      end
+
+      question = Question.find(question_id)
+
+      # Check for other open questions
+
+      question.sent_questions.create(
+        session: question.quiz.sessions.find_by(live: true),
```

You filter to live sessions only. Maybe you should also return an error when there are no live sessions for the question?

---

My plan is to come back and do a mass error-checking PR. At the moment, I'm just trying to get things working for the frontend.